README.md

```diff
@@ -99,6 +99,10 @@ Enter `localsettings.py` and do the following:
 - Find the `LOCAL_APPS` section and un-comment the line that starts with `'kombu.transport.django'`
 - You may also want to add the line `from dev_settings import *` at the top of the file, which includes some useful default settings.
 
+Create the shared directory.  If you have not modified `SHARED_DRIVE_ROOT`, then run:
```

Comment:
How about `settings.SHARED_DRIVE_ROOT`? I initially thought this might be an environment variable.

Reply:
Hmm I'm not sure if that effectively distinguishes between the variables defined in settings.py vs localsettings.py

I went with just the variable name to stay consistent with the conventions used previously, but would be prefer something that is more specific and not ambiguous.

Reply:
sounds reasonable